tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AspNetCore/DefaultModelBindingContext_SetResult_Integration.cs

```diff
@@ -0,0 +1,82 @@
+// <copyright file="DefaultModelBindingContext_SetResult_Integration.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#if !NETFRAMEWORK
+
+using System.ComponentModel;
+using Datadog.Trace.ClrProfiler.AutoInstrumentation.AspNetCore;
+using Datadog.Trace.ClrProfiler.CallTarget;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.DuckTyping;
+
+namespace Datadog.Trace.ClrProfiler.AspNetCore
+{
+    /// <summary>
+    /// setModel calltarget instrumentation
+    /// </summary>
+    [InstrumentMethod(
+    AssemblyName = "Microsoft.AspNetCore.Mvc.Core",
+    TypeName = "Microsoft.AspNetCore.Mvc.ModelBinding.DefaultModelBindingContext",
+    MethodName = "set_Result",
+    ReturnTypeName = ClrNames.Void,
+    ParameterTypeNames = new[] { "Microsoft.AspNetCore.Mvc.ModelBinding.ModelBindingResult" },
+    MinimumVersion = "2.1.16.0",
```

Comment:
Should this really be `2.2.0`? Did they really add it in `2.1.16`?!

Reply:
good question , actually I see that it was there from 2.0.0 
![image](https://user-images.githubusercontent.com/8353486/158226048-fa7b5039-5da8-4982-ac42-3d60f367d758.png)

Reply:
I changed it to 2.0.0.0 then, in https://github.com/DataDog/dd-trace-dotnet/pull/2495/commits/6785f56e8af6d600e01818116fa36b76c7e0cdb6